tests/Oryx.Integration.Tests/Database/DatabaseTestsBase.cs

```diff
@@ -81,6 +84,25 @@ protected DatabaseTestsBase(ITestOutputHelper outputHelper, Fixtures.DbContainer
                 });
         }
 
+        private List<EnvironmentVariable> getEnvironmentVariableList()
+        {
+            List<EnvironmentVariable> envVariableList = _dbFixture.GetCredentialsAsEnvVars();
+            var testStorageAccountUrl = Environment.GetEnvironmentVariable(SdkStorageConstants.TestingSdkStorageUrlKeyName);
+            var sdkStorageUrl = string.IsNullOrEmpty(testStorageAccountUrl) ? SdkStorageConstants.PrivateStagingSdkStorageBaseUrl : testStorageAccountUrl;
+            
+            envVariableList.Add(new EnvironmentVariable(SdkStorageConstants.SdkStorageBaseUrlKeyName, sdkStorageUrl));
+
+            if (sdkStorageUrl == SdkStorageConstants.PrivateStagingSdkStorageBaseUrl)
+            {
+                string stagingStorageSasToken = Environment.GetEnvironmentVariable(SdkStorageConstants.PrivateStagingStorageSasTokenKey) != null
```

Comment:
nit: same comment from the other file about using `??` in this case where we're only checking if the environment variable was set or not